dvc/command/pkg.py

```diff
@@ -54,36 +119,89 @@ def add_parser(subparsers, parent_parser):
         formatter_class=argparse.RawDescriptionHelpFormatter,
     )
     pkg_install_parser.add_argument(
-        "address",
-        nargs="?",
-        default="",
-        help="Package address: git://<url> or https://github.com/...",
+        "targets", nargs="*", default=[None], help="Package name."
     )
-    pkg_install_parser.add_argument(
-        "target_dir",
-        metavar="target",
-        nargs="?",
-        default=".",
-        help="Target directory to deploy package outputs. "
-        "Default value is the current dir.",
+    pkg_install_parser.set_defaults(func=CmdPkgInstall)
+
+    PKG_UNINSTALL_HELP = "Uninstall package(s)."
+    pkg_uninstall_parser = pkg_subparsers.add_parser(
+        "uninstall",
+        parents=[parent_config_parser, parent_parser],
+        description=append_doc_link(PKG_UNINSTALL_HELP, "pkg-uninstall"),
+        help=PKG_UNINSTALL_HELP,
+        formatter_class=argparse.RawDescriptionHelpFormatter,
     )
-    pkg_install_parser.add_argument(
-        "-s",
-        "--select",
-        metavar="OUT",
-        action="append",
-        default=[],
-        help="Select and persist only specified outputs from a package. "
-        "The parameter can be used multiple times. "
-        "All outputs will be selected by default.",
+    pkg_uninstall_parser.add_argument(
+        "targets", nargs="*", default=[None], help="Package name."
     )
-    pkg_install_parser.add_argument(
-        "-f",
-        "--file",
-        help="Specify name of the stage file. It should be "
-        "either 'Dvcfile' or have a '.dvc' suffix (e.g. "
-        "'prepare.dvc', 'clean.dvc', etc). "
-        "By default the file has 'mod_' prefix and imported package name "
-        "followed by .dvc",
+    pkg_uninstall_parser.set_defaults(func=CmdPkgUninstall)
+
+    PKG_ADD_HELP = "Add package."
+    pkg_add_parser = pkg_subparsers.add_parser(
+        "add",
```

Comment:
`add` command from the list `dvc pkg -h` looks a bit confusing. Why `install` is not enough?

Reply:
Because `install` is about git cloning package repo. `add` is about aliasing. You could
```
dvc pkg add mypkg https://github.com/iterative/mypkg # this is 
```
once for the project and then call
```
dvc pkg install mypkg
```
any time you want. Though with dynamic installs and fetches I'm not sure we even need `dvc pkg install` really.

Reply:
Probably, that install is for. It should be doing fetching and cloning. Not import.

Reply:
@shcheklein Fetching cache for all files for revision? That seems like it might be too much. Currently `import` is just fetching what it needs.

Reply:
import is a really strange place to do this, especially the way it was done in the __init__ of the dependency. We have commands `dvc fetch`, `dvc pull`, `dvc checkout` - why not use them to manage data. Alternatively, `pkg install` can traverse DVC-files (that in this case serve as a requirements.txt) and fetch only needed data. I think we had a few discussions about this.